tools/linter_lib/custom_check.py

```diff
@@ -212,6 +212,12 @@
             "good_lines": ["#my-style {color: blue;}"],
             "bad_lines": ['<p style="color: blue;">Foo</p>', 'style = "color: blue;"'],
         },
+        {
+            "pattern": r"assert\(.*\)",
```

Comment:
Just `r"assert\("`. There’s no reason to look for the right parenthesis, and that will fail for calls that wrap across multiple lines.

Reply:
Done!